uiowa/uiowa
# Feature: Add main navigation menu depth to config ignore.

Adds the following items to the config ignore settings:
block.block.main_navigation__superfish__horizontal:depth
block.block.main_navigation__superfish__toggle:depth
block.block.main_navigation__superfish__mega:depth

This allows developers to override the default menu depth to hide deep menus if we want them hidden, and the change wont get overwritten on the next release.

# How to test

drush @default.local cr
blt ds
drush @default.local uli
Change visible depth of any version of the main navigation from Unlimited to any number.
Navigate to Administration > Configuration > Development > Synchronize and make sure the change is in the ignored configuration entities (???)

This shows the reason for adding this capability;
On a wide screen display, the menu carries way over to the right hand side:
![Screen Shot 2021-03-08 at 2 55 20 PM](https://user-images.githubusercontent.com/3067017/110381218-c5bffd00-801e-11eb-9bb1-fc412bb8005f.png)

On a more traditional sized display, the menu overlaps its grandparent menu in this criss-cross effect:
![Screen Shot 2021-03-08 at 2 55 43 PM](https://user-images.githubusercontent.com/3067017/110381225-c8225700-801e-11eb-9c1a-c6c8a0c0d9eb.png)
